### PR TITLE
Add API version 3

### PR DESCRIPTION
**Pull request by DMahoney0**
This pull request adds version 3 of the API documentation for Metasys 11.0

Additional updates include:
- All previous resource references updated to v3
- A header section for the response payload schema section
- A header section for the response and request payloads using enumerations
- Adding v3 to the version dropdown as pre-release

### CODE REVIEW

File: _data/default/cards.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
       intro: 'Browse the documentation of our REST API'
       color: 'blue'
       icon:  'fa fa-file-alt'
-      url: 'api/v2/'
+      url: 'api/v3/'
```
Comment (mgwelch) [inline _data/default/cards.yml:14]: This should not default to v3 until it's released.

File: _data/default/api.yml [hunk 2]
```diff
@@ -1,4 +1,6 @@
-- name: '<i class="metasys"></i> API v2 (latest)'
+- name: '<i class="metasys"></i> API v3 (latest)'
```
Comment (mgwelch) [inline _data/default/api.yml:1]: This shouldn't me marked (latest) until released. Perhaps (pre-release) would be a good tag to add.

Comment (DMahoney0) [reply _data/default/api.yml]: Correct

File: _data/default/cards.yml (hunk 1 above)
Comment (DMahoney0) [reply _data/default/cards.yml]: Corrected

**Pull request merged**